main.py

```diff
@@ -1,7 +1,24 @@
 import init_django_orm  # noqa: F401
+from db.models import Genre, Actor
 
-from django.db.models import QuerySet
+def main() -> None:
+    Genre.objects.create(name="Western")
+    Genre.objects.create(name="Action")
+    Genre.objects.create(name="Dramma")
 
+    Actor.objects.create(first_name="George", last_name="Klooney")
+    Actor.objects.create(first_name="Kianu", last_name="Reaves")
```

Comment:
There is a typo in the actor's first name 'Kianu'. It should be 'Keanu'. This is corrected later in the code, but it's better to avoid creating incorrect entries initially.